src/components/wizard/wizard.component.spec.ts

```diff
@@ -35,11 +33,20 @@ export class WizardAsyncValidationTestComponent {
     firstStepValidator: () => boolean | Promise<boolean>;
     lastStepValidator: () => boolean | Promise<boolean>;
 
-    stepChanging(_: StepChangingEvent) {}
-    stepChange(_: number) {}
-    onNext(_: number) {}
-    onFinishing() {}
-    onFinish() {}
+    stepChanging(_: StepChangingEvent) {
+    }
```

Comment:
I really dislike this auto formatting. Could you reconfigure your formatter to ignore lines like this, and undo these unnecessary changes?

Reply:
I think this was probably bad configuration in my editor 🤦‍♂️ sorry Darren

Reply:
No problem @ashley-hunter, I'll back that out in my next commit.